### ISSUE TITLE
Incorrectly removes double curly braces in f-string

### ISSUE THREAD

**Issue by johanneskoester** [snakemake/snakefmt#215]
The following:
```python
def get_test_regions(wildcards):
    benchmark = config["variant-calls"][wildcards.callset]["benchmark"]
    return f"resources/regions/{benchmark}/test-regions.cov-{{cov}}.bed"
```
get incorrectly reformatted to

```python
def get_test_regions(wildcards):
    benchmark = config["variant-calls"][wildcards.callset]["benchmark"]
    return f"resources/regions/{benchmark}/test-regions.cov-{cov}.bed"
```

(double curly braces are removed)
This is a pretty severe bug as it alters the logic of the code and leads to unexpected errors.

**Comment by mbhall88** [comment]
This isn't precisely a snakefmt problem. I relates to #207 and python the way in which python 3.12 tokenizes f-strings (this is formatted correctly in python 3.10).

Nevertheless, it is a bug. I'll try and get on to it ASAP. I started a discussion in the development channel on the snakemake discord about the future of dealing with bugs etc in this repo and it would be good to have your input @johanneskoester

**Comment by johanneskoester** [comment]
Somebody who can probably give valuable input here could be @Hocnonsense who fixed f-string handling for Python 3.12 in Snakemake itself.
